packages/web/src/pages/upload-page/fields/TrackMetadataFields.tsx

```diff
@@ -56,7 +56,13 @@ export const TrackMetadataFields = (props: TrackMetadataFieldsProps) => {
             aria-label={messages.genre}
             placeholder={messages.genre}
             mount='parent'
-            menu={{ items: GENRES }}
+            menu={{
+              items: GENRES.map((g) => ({
+                el: g,
+                text: g,
+                value: convertGenreLabelToValue(g)
```

Comment:
we need these inputs to be controlled for multi-track to work, so I think we'd need to call `convertGenreLabelToValue` on the field value inside `DropdownField` for it to map properly to this new menu value?

Reply:
ohh got it ok. ill just revert and let you do it the right way lol